libs/market-depth/src/lib/orderbook.tsx

```diff
@@ -313,40 +365,78 @@ export const Orderbook = ({
   const c = theme === 'dark' ? colors.neutral[600] : colors.neutral[300];
   const gradientStyles = `linear-gradient(${c},${c}) 24.6% 0/1px 100% no-repeat, linear-gradient(${c},${c}) 50% 0/1px 100% no-repeat, linear-gradient(${c},${c}) 75.2% 0/1px 100% no-repeat`;
 
+  const resolutions = new Array(decimalPlaces + 1)
+    .fill(null)
+    .map((v, i) => Math.pow(10, i));
+
   return (
-    <div
-      className={`h-full overflow-auto relative ${styles['scroll']} pl-2 text-xs`}
-      onScroll={onScroll}
-      ref={scrollElement}
-      data-testid="scroll"
-    >
+    <div className="h-full relative pl-2 text-xs">
       <div
-        className="sticky top-0 grid grid-cols-4 gap-2 text-right border-b pt-2 bg-white dark:bg-black z-10 border-default"
+        className="absolute top-0 grid grid-cols-4 gap-2 text-right border-b pt-2 bg-white dark:bg-black z-10 border-default w-full"
         style={{ gridAutoRows: '17px' }}
+        ref={headerElement}
       >
         <div>{t('Bid vol')}</div>
         <div>{t('Price')}</div>
         <div>{t('Ask vol')}</div>
         <div className="pr-[2px]">{t('Cumulative vol')}</div>
       </div>
       <div
-        className="relative text-right"
-        style={{
-          paddingTop: `${paddingTop}px`,
-          paddingBottom: `${paddingBottom}px`,
-          minHeight: `calc(100% - ${2 * (rowHeight + 2)}px)`,
-          background: tableBody ? gradientStyles : 'none',
-        }}
+        className={`h-full overflow-auto relative ${styles['scroll']} pt-[26px] pb-[17px]`}
+        onScroll={onScroll}
+        ref={scrollElement}
+        data-testid="scroll"
       >
-        {tableBody || (
-          <div className="inset-0 absolute">
-            <Splash>{t('No data')}</Splash>
-          </div>
-        )}
+        <div
+          className="relative text-right min-h-full"
+          style={{
+            paddingTop: `${paddingTop}px`,
```

Comment:
chore (blocking): no need to add 'px'

```suggestion
            paddingTop: paddingTop,
```